### ISSUE TITLE
我在WSSS4LUAD的数据集挑战赛中看到更高的分数，评价规则也是mIOU

### ISSUE THREAD

**Issue by 464hee** [Vison307/PistoSeg#2]
他们的分数已经达到了0.84，请问相比于论文实验使用对比数据中的mIOU相差较大，可以解释一下吗，这个是挑战链接：https://wsss4luad.grand-challenge.org/，排行榜中有得分数据。

**Comment by Vison307** [comment]
1. 本文提出了一种新的特征一致性约束，实现不同方法（即logits和CAMs）分割结果的融合，这是一个不局限于计算病理学的通用框架。
2. 此外，通过提高overlapping ratio，进行形态学膨胀、腐蚀等后处理，以及模型集成等方式，PistoSeg的性能可以进一步提高到mIoU>0.80。事实上，我们是这项挑战的top队伍之一。
---
1. This paper proposes a novel feature consistency constraint to realize the fusion of segmentation results of different methods (i.e., logits and CAMs), which is a general framework not limited to pathology. 
2. Furthermore, with an increased overlap ratio, post-processing like morphology dilation, and model ensemble, PistoSeg can also reach mIoU>0.80. In fact, we are one of the top teams in the challenge.

若有其他问题请重开issue，谢谢。
